assets/js/utils/transforms.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+import { createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import sharedAttributes from './shared-attributes';
+
+export function makeTransform( toBlock, columns, defaultAttributes ) {
```

Comment:
This `columns` variable threw me at first since I expected it was related to the attribute `columns` - but I'm not sure you need it? I think you could get rid of `makeSharedAttributesTransform` entirely, [since `createBlock` will only use attributes defined in the block schema.](https://github.com/WordPress/gutenberg/blob/211f65dd854fc1f7d7e54782863ad0629abeff41/packages/blocks/src/api/factory.js#L39)

Actually, I think you can use createBlock directly in the transform, and then you don't even need this helper file. Like, in newest products:

```js
transform: ( attributes ) => createBlock( 'woocommerce/product-new', attributes )
```